recording_server/src/utils/system-monitor.ts

```
@@ -0,0 +1,260 @@
+        }
+    }
+
+    async logStatsToFile(): Promise<void> {
+        try {
+            const stats = await this.collectStats()
+            await fs.promises.appendFile(
+                this.logPath,
+                JSON.stringify(stats) + '\n',
+            )
+        } catch (error: any) {
+            logger.error('Error collecting stats: ' + error.message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)